templates/datastore.html

```diff
@@ -0,0 +1,301 @@
+                        <input type="text" class="form-control" id="datastore_name" name="datastore_name">
+                        <label for="destination_path">Destination Path:</label>
+                        <input type="text" class="form-control" id="destination_path" name="destination_path">
+                    </div>
+                    <button type="submit" class="btn btn-primary" name="add_datastore">Add FileSystem</button>
+                </div>
+
+                <!-- MongoDB Configuration -->
+                <div id="mongodbForm" class="hidden">
+                    <h4>MongoDB Configuration</h4>
+                    <div class="mb-3">
+                        <label for="datastore_name_mongo">Datastore Name:</label>
+                        <input type="text" class="form-control" id="datastore_name_mongo" name="datastore_name_mongo">
+                    </div>
+                    <div class="mb-3">
+                        <label for="username_mongo">Username:</label>
+                        <input type="text" class="form-control" id="username_mongo" name="username_mongo">
+                    </div>
+                    <div class="mb-3">
+                        <label for="password_mongo">Password:</label>
+                        <input type="text" class="form-control" id="password_mongo" name="password_mongo">
+                    </div>
+                    <div class="mb-3">
+                        <label for="host_mongo">Host:</label>
+                        <input type="text" class="form-control" id="host_mongo" name="host_mongo">
+                    </div>
+                    <div class="mb-3">
+                        <label for="port_mongo">Port:</label>
+                        <input type="text" class="form-control" id="port_mongo" name="port_mongo">
+                    </div>
+                    <div class="mb-3">
+                        <label for="database_mongo">Database:</label>
+                        <input type="text" class="form-control" id="database_mongo" name="database_mongo">
+                    </div>
+                    <button type="submit" class="btn btn-primary" name="add_datastore">Add MongoDB</button>
+                </div>
+            </form>
+        </div>
+
+        <!-- Existing DataStores -->
+        <table class="table table-striped">
+            <thead>
+                <tr>
+                    <th>Datastore ID</th>
+                    <th>Datastore Name</th>
+                    <th>Datastore Type</th>
+                    <th>Date Created</th>
+                    <th>Remove Datastore</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for datastore in datastores %}
+                <tr>
+                    <td>{{ datastore.UUID }}</td>
+                    <td>{{ datastore.Name }}</td>
+                    <td>{{ datastore.Type }}</td>
+                    <td>{{ datastore.Created }}</td>
```

Comment:
Need to add the connected column.

Reply:
Added the functionality to connect/disconnect form here.